finetuner-workflow/finetuner/finetuner.py

```diff
@@ -493,6 +500,26 @@ def compute_loss(self, model, inputs, return_outputs=False):
         return results
 
 
+def tensorizer_save(checkpoint_dir: str) -> None:
+    checkpoint_model = trainer.model
```

Comment:
The model should probably be an argument instead of pulling it from the `trainer` global.